docs/migration-guide-java.md

```diff
@@ -0,0 +1,157 @@
+---
+---
+
+# Migration guide (Java)
+Welcome!
+
+As you may already know RAML 0.8/1.0 Java parser `raml-java-parser` has been deprecated in favor of the new and better one - `webapi-parser`. This guide describes how to migrate an existing code from `raml-java-parser` to `webapi-parser`.
```

Comment:
> in favor of the new and better one

let's avoid qualitative adjectives. we can simply say "in favor of `webapi-parser`"

Reply:
Fixed.